state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -347,6 +387,12 @@ pub mod pallet {
 			amount: AssetAmount,
 			fee: AssetAmount,
 		},
+		RefundEgressScheduled {
+			swap_id: SwapId,
+			egress_id: EgressId,
```

Comment:
Has been discussed in the notion document. We don't need to repeat fields from other events, the product can look them up by swap_id.

Reply:
we dont need the asset on this, but i think in this case it would nice to have it. eg it makes it easier to index the events for some other reason (eg. monitoring ingresses and egresses) and it would be consistent with `SwapEgressScheduled`

Reply:
We've been talking about "normalizing" events, which I understand I "removing duplication". It wouldn't make sense if we emit two events under the same id but have different output assets, for example. I don't have a strong opinion here, happy to provide the asset here if it is helpful. My impression from the previous discussion (on the notion document) was that this didn't make any difference, and decided against duplicating it (which is how it's been shown in the notion document).

Reply:
Added asset now